all-the-icons.el

```diff
@@ -557,7 +560,54 @@ When F is provided, the info function is calculated with the format
 
   (defun all-the-icons--data-name (name)
     "Get the symbol for an icon family function for icon set NAME."
-    (intern (concat "all-the-icons-" (downcase (symbol-name name)) "-data"))))
+    (intern (concat "all-the-icons-" (downcase (symbol-name name)) "-data")))
+
+  (defun all-the-icons--insert-function-name (name)
+    "Get the symbol for an icon insert function for icon set NAME."
+    (intern (concat "all-the-icons-" (downcase (symbol-name name)) "-insert"))))
```

Comment:
Personally, I would prefer the function name to be `all-the-icons-insert-<fontset>`, _e.g._ `all-the-icons-insert-faicon` or `all-the-icons-insert-wicon`

Soo, I guess this would be
```elisp
(concat "all-the-icons-insert" (downcase (symbol-name name)))
```
The API becomes inconsistent between like `all-the-icons-<fontset>-data` and `all-the-icons-<fontset>-family`, but since the insert functions are interactive I don't mind this :)